nav2_smac_planner/include/nav2_smac_planner/utils.hpp

```diff
@@ -104,6 +104,36 @@ inline double findCircumscribedCost(std::shared_ptr<nav2_costmap_2d::Costmap2DRO
   return result;
 }
 
+/**
+ * @brief Checks for the cusp(s) position(s)
+ * @param path Path in which to look for cusps
+ * @return Cusps' indices plus initial and final path poses' indices
+ */
+inline std::vector<unsigned int> findDirectionChange(const nav_msgs::msg::Path & path)
```

Comment:
Need a test for this

Reply:
Where should the test be added? test_smac_hybrid, test_nodehybrid or somewhere else?

Reply:
You can add a new file `test_utils.cpp` if you like